vtipnik.py

```diff
@@ -18,16 +19,24 @@ def seznam_vtipu():
     """
     Funkce načtě obsah složky a vrátí seznam všech dostupných vtipů
     """
-    return glob("vtipy/*")
+    return glob.glob("vtipy/*")
 
 
 def main():
     """
-    Vypíše všechny nalezené vtipy.
+    Vypíše jeden náhodný vtip ze seznamu a po stisknutí Enteru vypíše další
     """
-    for vtip in seznam_vtipu():
-        print("=" * 80)
-        print(nacti_vtip(vtip))
+    ls_vtip=seznam_vtipu()
+    for x in range(len(ls_vtip)):
+       print("=" * 80) 
+       vtip=random.choice(ls_vtip)
```

Comment:
Prosim pridat mezery kolem binarnich operatoru podle PEP8.
```suggestion
       vtip = random.choice(ls_vtip)
```